Merian/Wband.yaml

```diff
@@ -0,0 +1,17 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  daily:
```

Comment:
I am wondering if `daily` is the most distinctive term for that dataset. I could imagine that there could also be `daily` raw files, `daily` files of a further processing level etc... in the future. Our goal with this catalog is that identifiers which have once been issued will ideally never change again, thus, we'd like to be careful when choosing them.

I could imagine `motion_corrected` or `daily_motion_corrected` or `motion_corrected.daily` would be more specific. I tend to opt for `motion_corrected` and skip `daily` as we didn't specify the time interval for other datasets either, but maybe this one is different, as there exist daily and hourly files.

Reply:
I put motion_corrected, I think that's the best